content/realtime/usage.textile

```diff
@@ -85,9 +85,21 @@ blang[ruby].
   ```
 
 blang[java].
-  The Realtime library for Java and Android is "hosted on Github":https://github.com/ably/ably-java and is downloadable as a JAR from "https://github.com/ably/ably-java/releases":https://github.com/ably/ably-java/releases or via "JCentre":https://bintray.com/ably-io/ably.
+  The Realtime library for Java and Android is "hosted on Github":https://github.com/ably/ably-java and can be used by adding one line to <span language="default">@build.gradle@ dependencies section.
 
-  Ensure the JAR is included in your classpath as follows:
+  For Java applications:
+
+  ```[java]
+  compile 'io.ably:ably-java:0.8.7'
+  ```
+
+  For Android apps:
+
+  ```[java]
+  compile 'io.ably:ably-android:0.8.7'
+  ```
+
+  Ensure the library is included in your classpath as follows:
```

Comment:
That's fine. I don't think the docs and tutorials always have to point to the latest, but equally we'll need to ensure they're reasonably up to date